patterns/blueprint-vpc-lattice/cluster1/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "configure_kubectl" {
+  description = "Configure kubectl: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
+  value       = "aws eks update-kubeconfig --name ${module.eks.cluster_name} --alias ${module.eks.cluster_name} --region ${local.region}"
```

Comment:
please copy the output from the other patterns - all of the boilerplate configurations/instructions should be consistent across the patterns